dist/origin-web-catalogs.css

```diff
@@ -60,37 +60,56 @@
   padding-bottom: 40px;
 }
 .landing-main-area {
-  margin: 15px 320px 15px 20px;
+  margin: 15px 20px;
   padding-right: 10px;
 }
+@media (min-width: 768px) {
```

Comment:
Should we be using constants for this?  I noticed them here:  https://github.com/openshift/origin-web-console/blob/master/app/scripts/app.js#L435